src/directions/distance_matrix_service.ts

```diff
@@ -16,19 +18,15 @@ import {
   formatSecondsAsGoogleDurationText,
   parseOrFindLocations,
 } from "./helpers";
+import { createBoundsFromPositions } from "../common";
+import { LngLat } from "maplibre-gl";
 
 // formatted_address needed for originAddresses and destinationAddresses
 const DISTANCE_MATRIX_FIND_LOCATION_FIELDS = ["geometry", "formatted_address"];
 const KILOMETERS_TO_METERS_CONSTANT = 1000;
 
 export class MigrationDistanceMatrixService {
-  // This will be populated by the top level module
-  // that creates our location client
-  _client: LocationClient;
-
-  // This will be populated by the top level module
```

Comment:
Keep just the single line comment: `// This will be populated by the top level module`